sections/main-product.liquid

```diff
@@ -504,6 +504,171 @@
         </a>
       </div>
     </div>
+    {%- if section.settings.media_position == 'right' -%}
```

Comment:
I think we'll need to follow this same approach on `featured-product` now that we removed `product--reverse` class. Otherwise, the setting won't do anything.